invenio_app_rdm/config.py

```diff
@@ -430,6 +430,17 @@ def files_rest_permission_factory(obj, action):
         'schema': 'http://www.openarchives.org/OAI/2.0/oai_dc.xsd',
         'namespace': 'http://www.openarchives.org/OAI/2.0/oai_dc/',
     },
+    'datacite': {
+        'serializer': 'invenio_rdm_records.oai:datacite_etree',
+        'schema': 'http://schema.datacite.org/'
+                  'meta/nonexistant/nonexistant.xsd',
```

Comment:
Not sure what is better, breaking the line like this or adding a `# noqa` so it passe PEP 8... In the end is for aesthetics but is harder to read the line when separated. WDYT @mitsosf ? The alternative would be:

```suggestion
        'schema': 'http://schema.datacite.org/meta/nonexistant/nonexistant.xsd',  # noqa
```